pkg/render/apiserver.go

```diff
@@ -2044,7 +2044,7 @@ func (c *apiServerComponent) tieredPolicyPassthruClusterRole() *rbacv1.ClusterRo
 	return &rbacv1.ClusterRole{
 		TypeMeta: metav1.TypeMeta{Kind: "ClusterRole", APIVersion: "rbac.authorization.k8s.io/v1"},
 		ObjectMeta: metav1.ObjectMeta{
-			Name: "calico-tiered-policy-passthrough",
+			Name: "calico-ee-tiered-policy-passthrough",
```

Comment:
@caseydavenport  @rene-dekker  We have a [calico-tiered-policy-passthrough](https://github.com/tigera/operator/blob/release-v1.38/pkg/render/apiserver.go#L2100) ClusterRole and ClusterRoleBinding in globalObjects, which grant RBAC permissions for networkpolicies and globalnetworkpolicies. For enterprise, globalEnterpriseObjects  had a [tigera-tiered-policy-passthrough](https://github.com/tigera/operator/blob/release-v1.38/pkg/render/apiserver.go#L2145) ClusterRole and ClusterRoleBinding that provide access to stagednetworkpolicies and stagedglobalnetworkpolicies. While standardizing the naming prefix, I accidentally reused the same name, which appears to have overwritten the calico-tiered-* policy. To resolve this, I renamed the enterprise policy to calico-ee-tiered-policy-passthrough.